### PR TITLE
9.4_vlad_order_app

### PR DESCRIPTION
**Pull request by Kryvovyaz**
still have some issue with checkBox, when you uncheck it Boolean is stays true. Any help will be appreciated.

### CODE REVIEW

File: 9.4_Vlad_OrderMenu/app/src/main/java/com/example/a92_vlad_ordermenu/OrderViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    var totalCountLiveData = MutableLiveData<Int>()
+    var nameLiveData = MutableLiveData<String>()
+    var costAddWhippedCream = 1.50
+    var costAddChocolate = 2.75
+    var totalCost = 0.0
+    var costBasic: Double = 5.00
+
+    fun increaseQuantity() {
+        totalCountLiveData.value = (totalCountLiveData.value)?.plus(1)
+
```
Comment (oliver-basoglu) [inline 9.4_Vlad_OrderMenu/app/src/main/java/com/example/a92_vlad_ordermenu/OrderViewModel.kt:20]: unnecessary new line. Please also update other files, there a couple more

File: 9.4_Vlad_OrderMenu/app/src/main/java/com/example/a92_vlad_ordermenu/OrderViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        isCreamLiveData.value = false
+        isCholateLiveData.value = false
+        nameLiveData.value = ""
+        return """
+        
+                    Add whipped cream? ${isCreamLiveData.value}
+                    Add chocolate? ${isCholateLiveData.value}
+                    Quantity: ${totalCountLiveData.value}
+                    Total is: $${calculateTotal()}
+                """.trimIndent()
```
Comment (oliver-basoglu) [inline 9.4_Vlad_OrderMenu/app/src/main/java/com/example/a92_vlad_ordermenu/OrderViewModel.kt:41]: Nice job

File: 9.4_Vlad_OrderMenu/app/src/main/java/com/example/a92_vlad_ordermenu/OrderViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    fun calculateTotal(): Double {
+        var tempcream = 0.0
+        var tempchocolate = 0.0
+        if (isCreamLiveData.value == true) {
+            tempcream = costAddWhippedCream * totalCountLiveData.value!!
```
Comment (oliver-basoglu) [inline 9.4_Vlad_OrderMenu/app/src/main/java/com/example/a92_vlad_ordermenu/OrderViewModel.kt:48]: Add init block so you can avoid using !!

init{
totalCountLiveData.value = 0
totalCountLiveData.value = 0
}

Comment (Kryvovyaz) [reply 9.4_Vlad_OrderMenu/app/src/main/java/com/example/a92_vlad_ordermenu/OrderViewModel.kt:48]: compiler still shows an error after I inserted init block

**Pull request merged**